flink-connector-oceanbase-tools-cdc/src/main/java/com/oceanbase/connector/flink/tools/catalog/OceanBaseSchemaFactory.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2024 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.connector.flink.tools.catalog; // Licensed to the Apache Software Foundation
+
+import org.apache.flink.util.Preconditions;
+
+import org.apache.commons.collections.CollectionUtils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+/**
+ * Factory that creates oceanbase schema.
+ *
+ * <p>In the case where oceanbase schema needs to be created, it is best to create it through this
+ * factory
+ */
+public class OceanBaseSchemaFactory {
+
+    public static TableSchema createTableSchema(
+            String database,
+            String table,
+            Map<String, FieldSchema> columnFields,
+            List<String> pkKeys,
+            String tableComment) {
+        TableSchema tableSchema = new TableSchema();
+        tableSchema.setDatabase(database);
+        tableSchema.setTable(table);
+        tableSchema.setFields(columnFields);
+        tableSchema.setKeys(buildKeys(pkKeys, columnFields));
+        tableSchema.setTableComment(tableComment);
+        tableSchema.setDistributeKeys(buildDistributeKeys(pkKeys, columnFields));
+        return tableSchema;
+    }
+
+    private static List<String> buildDistributeKeys(
```

Comment:
What does DistributeKeys mean in oceanbase, oceanbase only has index keys and partition keys.